test/e2e/dataplane/tcp_pod_to_service.go

```diff
@@ -10,50 +10,37 @@ import (
 )
 
 var _ = Describe("[dataplane] Basic Pod to Service tests across clusters without discovery", func() {
-
 	f := framework.NewDefaultFramework("dataplane-p2s-nd")
 
-	It("Should be able to perform a Pod to Service TCP connection and exchange data between different clusters NonGW to NonGW", func() {
-		testPod2ServiceTCP(f, framework.NonGatewayNode, framework.NonGatewayNode)
-	})
+	verifyInteraction := func(listenerScheduling, connectorScheduling framework.NetworkPodScheduling) {
```

Comment:
Please rebase the PR as few changes were merged recently that affects this file.

Reply:
Done